doc/Makefile

```diff
@@ -1,30 +1,31 @@
-#
-# Top level makefile for documentation build
-#
+# ------------------------------------------------------------------------------
+# Makefile for documentation build
+# SPDX-License-Identifier: Apache-2.0
 
 BUILDDIR ?= _build
 DOC_TAG ?= development
 SPHINXOPTS ?= -j auto
 KCONFIG_TURBO_MODE ?= 0
 
+# ------------------------------------------------------------------------------
 # Documentation targets
-# ---------------------------------------------------------------------------
 
-.PHONY: clean htmldocs htmldocs-fast pdfdocs doxygen
+.PHONY: configure clean html html-fast latex pdf doxygen
 
-htmldocs-fast:
-	${MAKE} htmldocs KCONFIG_TURBO_MODE=1
+html-fast:
+	${MAKE} html KCONFIG_TURBO_MODE=1
 
-htmldocs pdfdocs doxygen: configure
-	cmake --build ${BUILDDIR} -- $@  # -v # VERBOSE=1
+html latex pdf doxygen: configure
+	cmake --build ${BUILDDIR} --target $@
 
-# Run CMake every time cause it's quick and re-configures TURBO_MODE if
-# needed
-.PHONY: configure
 configure:
-	cmake -GNinja -B${BUILDDIR} -S. -DDOC_TAG=${DOC_TAG} \
+	cmake \
+		-GNinja \
```

Comment:
Users are using `make` to invoke this Makefile which then invokes CMake, where Ninja is requested as generator.
As user is already using make in first place, then why not use a Makefile generator ?

Note, no change requested here, just me being curious.

Reply:
I'm not sure to be honest. This Makefile is just a wrapper around CMake, which should be deleted IMO. It was agreed to keep it, though.

Reply:
and that is fully accepted, it just looks a bit funny to me to use make to invoke ninja. :laughing: